addons/event_sale/tests/test_event_sale.py

```diff
@@ -482,6 +482,50 @@ def test_ticket_price_with_pricelist_and_tax(self):
         })
         self.assertEqual(so.amount_total, 660.0, "Ticket is $1000 but the event product is on a pricelist 10 -> 6. So, $600 + a 10% tax.")
 
+    def test_event_sale_stat_button(self):
+        """ Test that the amount on the sale stat button should reflect the total price of only the confirmed orders. """
+        # Confirming existing SO with two lines
+        ticket = self.event_0.event_ticket_ids[0]
+        sale_order_line = self.env['sale.order.line'].create({
```

Comment:
I think we can create all SOL and registration at once with multi-create
And then perform actions.